contracts/payment/ILemonadeEscrow.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+struct RefundPolicy {
+    uint32 timestamp;
```

Comment:
You might want to consider using uint256 as a timestamp to allow for easy comparison with block.timestamp.